### PR TITLE
Fix a few typos

### PR DESCRIPTION
**Pull request by AlD**

### CODE REVIEW

File: docs/manuals/source/Configuration/Director.rst [hunk 1]
```diff
@@ -268,7 +268,7 @@ Use :config:option:`dir/FileSet/IgnoreFileSetChanges` to disable this behavior.
 
 .. _fileset-include:
 
-FileSet Include Ressource
+FileSet Include Resource
 ~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (fbergkemper) [inline docs/manuals/source/Configuration/Director.rst:272]:
```suggestion
~~~~~~~~~~~~~~~~~~~~~~~~
```

File: docs/manuals/source/Configuration/Director.rst [hunk 2]
```diff
@@ -518,13 +518,13 @@ Please take note of the following items in the FileSet syntax:
 
 .. _fileset-exclude:
 
-FileSet Exclude Ressource
+FileSet Exclude Resource
 ~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (fbergkemper) [inline docs/manuals/source/Configuration/Director.rst:522]:
```suggestion
~~~~~~~~~~~~~~~~~~~~~~~~
```

File: docs/manuals/source/Configuration/Director.rst [hunk 3]
```diff
@@ -567,7 +567,7 @@ FileSet Exclude-Ressources very similar to Include-Ressources, except that they
 
 .. _fileset-options:
 
-FileSet Options Ressource
+FileSet Options Resource
 ~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (fbergkemper) [inline docs/manuals/source/Configuration/Director.rst:571]:
```suggestion
~~~~~~~~~~~~~~~~~~~~~~~~
```

**Comment by fbergkemper** [comment]
Thanks for the PR. Looks good. However, I made some change requests, please see above.

**Comment by AlD** [comment]
Incorporated the suggestions.

**Comment by franku** [comment]
Thank you for your contribution! 

I would like to ask for two things; 

1. Please rewrite your commit message to something like: 
`docs: correct some typos`

2. Please add your name in the AUTHOR file at the root directory of bareos source. 

Thank you!

**Comment by AlD** [comment]
Done.

**Pull request merged**